.github/workflows/secrets.yml

```diff
@@ -0,0 +1,24 @@
+name: Secrets and Environment Variables
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
@lucasmiranda0, can we adjust to run this only when new pulls are sent to my GitHub account?